src/components/ebay-menu-button/template.marko

```diff
@@ -1,25 +1,38 @@
+<script marko-init>
+    var processHtmlAttributes = require("../../common/html-attributes");
+</script>
+
+<var isFake=(data.type === 'fake')/>
+<var baseClass=(isFake ? 'fake-menu-button' : 'menu-button')/>
+
 <span
-    class=data.menuClass
+    class=[
+        (isFake ? 'fake-menu-button' : 'menu-button'),
+        data.class
+    ]
     style=data.style
     w-bind
     w-on-expander-expand="handleExpand"
     w-on-expander-collapse="handleCollapse"
-    ${data.htmlAttributes}>
+    ${processHtmlAttributes(data)}>
     <ebay-button
         w-id="button"
-        class=data.buttonClass
+        class=[
+            "${baseClass}__button",
+            (data.borderless && 'expand-btn--borderless')
+        ]
         variant="expand"
         size=data.size
         priority=data.priority
-        no-text=data.noText
-        aria-expanded=String(data.expanded)
+        no-text=(!data.text && !data.icon)
+        aria-expanded="false"
```

Comment:
Is this always expanded when loaded now?  I see there is an `expanded` boolean in the attributes that are passed in.  Is that still an option?

Reply:
@msendlakowski it was always collapsed when loaded previously, but the aria attribute was off.

I brought up to @seangates that we should fix this and expose an `expanded` attribute (removed in this PR because it wasn't doing anything) that is actually linked to the expander instance.